genestack_client/scripts/genestack_user_setup.py

```diff
@@ -38,6 +38,52 @@ def ask_alias(existed):
         return alias
 
 
+def ask_use_token():
```

Comment:
Call this `select_auth_method`.

Reply:
`ask_` is common prefix for such typo of function, should we?